### PR TITLE
fix hive feature derivation

### LINKED ISSUE #1232: Exception when running without  COLUMN specified in SQL
**Description**
Exception when running without  COLUMN specified in SQL

**Reproduction Steps**
```
%%sqlflow
SELECT *
FROM notebook_dev.iris_train
TO TRAIN sqlflow_models.DNNClassifier
WITH
    model.n_classes = 3,
    model.hidden_units = [10,40],
    train.epoch = 3,
    train.batch_size = 64
LABEL class
INTO notebook_dev.iris_dnn_model;
```

**Expected Behavior**
It should behave same as when running with specified COLUMNS 'COLUMN sepal_width...'

**Screenshots**
```
_Rendezvous: <_Rendezvous of RPC that terminated with:
	status = StatusCode.UNKNOWN
	details = "column not found or inferred: sepal_width"
	debug_error_string = "{"created":"@1574413706.534090138","description":"Error received from peer ipv4:10.82.128.7:8005","file":"src/core/lib/surface/call.cc","file_line":1055,"grpc_message":"column not found or inferred: sepal_width","grpc_status":2}"
```

**Environment (Please complete the following information):**
 - OS:
 - Browser:
 - Version:

**Additional Notes**

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/sql-machine-learning/sqlflow/issues/1232

### CODE REVIEW

File: pkg/sql/feature_derivation.go [hunk 1]
```diff
@@ -270,7 +274,10 @@ func InferFeatureColumns(ir *codegen.TrainIR) error {
 		if err != nil {
 			return err
 		}
-		fillFieldMeta(columnTypes, rowData, fmMap)
+		err = fillFieldMeta(columnTypes, rowData, fmMap)
```
Comment (tonyyang-svail) [inline pkg/sql/feature_derivation.go:278]: Lesson learned: always handle error value.

**Review by tonyyang-svail — COMMENTED** [review]
Hi @typhoonzero, I also spotted the same error during #1251. And I filed a fix at #1253. Please feel free to use either this PR or #1253. :)

**Comment by typhoonzero** [comment]
Merging https://github.com/sql-machine-learning/sqlflow/pull/1253

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**